package.json

```diff
@@ -18,6 +18,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-lottie": "^1.2.3",
+    "sharp": "^0.32.1",
```

Comment:
are we actually using this module ? if not please uninstall

Reply:
Sharp is for image optimization in production internally used by Next js